caom2utils/caom2utils/fits2caom2.py

```diff
@@ -2806,10 +2842,10 @@ def _get_requirements(self):
         self.logger.debug('Begin Requirement augmentation.')
         flag = self._get_from_list('Observation.requirements.flag', index=0)
         self.logger.debug('End Requirement augmentation.')
+        reqts = None
         if flag:
-            return Requirements(flag)
-        else:
-            return None
+            reqts = Requirements(flag)
+        return reqts
```

Comment:
Better yet, it could be turned into a one-liner:
`return Requirements(flag) if flag else None`
There are a couple of other places where this applies

Reply:
Should be replaced everywhere.